docs/root/configuration/http/http_conn_man/traffic_splitting.rst

```diff
@@ -109,6 +109,7 @@ to each upstream cluster.
          - match: { prefix: / }
            route:
              weighted_clusters:
+             runtime_key_prefix: "routing.traffic_split.helloworld"
```

Comment:
I think the indent here is probably wrong?

Reply:
You're right - fixed.